opentelemetry-api/src/opentelemetry/metrics/_internal/__init__.py

```diff
@@ -729,7 +730,7 @@ def get_meter(
     """
     if meter_provider is None:
         meter_provider = get_meter_provider()
-    return meter_provider.get_meter(name, version)
+    return meter_provider.get_meter(name, version, schema_url)
```

Comment:
Is it possible to add schema_url for logging api [get_logger](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/_logs/_internal/__init__.py#L214) as well?

Reply:
Based on @aabmass comment below, I removed get_meter changes and created a follow up issue #3451. It's still possible to set schema URL with other get_meter overloads on the provider directly, so it should not be a blocker for semconv changes.